### ISSUE TITLE
IA Request [Authenticated Experience, Preferred Medical Facilities in VA.gov profile]

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#23288]
## What this is:
Work with VSP Information Architecture experts to finalize the IA of your product for VA.gov

## When to use this: 
After you have received a response to your Collaboration Cycle kick-off ticket that indicates Information Architect Request is needed. 

## Requested artifacts:
*Provide artifacts as available. If not available at the time this issue is created, please add information as it becomes available.*

- Link to [Preferred Medical Facilities in VA.gov profile product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/preferred-facility) 
- Link to [mockups](https://preview.uxpin.com/bb87d0fa61a32938a47e7bcdc836db235ab77576#/pages/137989653/documentation/sitemap?mode=i)
- Link to [content source of truth](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/preferred-facility/frontend/design-specs.md)

## Information Architecture worksheet:
*Provide any the following information, as available, to help provide direction and support of your IA.*

**Content/feature placement** - Where do you believe this content and/or feature should live within VA.gov and why?
- On Profile within the authenticated experience. Unauthenticated users won't be able to access this. 

**Keywords and SEO** - What keywords does the primary audience use when they talk about or search for your content?
- We don't anticipate users searching for this feature specifically but plan to validate that in research.

**URL(s)** - What are your suggested URLs and why?
- Follow standard/existing convention leverage lower case and corresponding h1 on the page : profile/your-va-health-facilities

**Primary entry points** - What do you see as the primary entry point for users to find your new content/tool?
- To be validated in research but we expect via Profile and VAOS scheduling tool

**Crosslinking opportunities** - What additional related areas of the site would links to your content/tool be helpful?
- To be validated in research but initial thinking is from within the VAOS scheduling tool. May also be helpful to reference this section of profile on va.gov/health-care/schedule-view-va-appointments/

**Redirects** - Will any existing content and/or tools need to be retired and redirected the new feature?
- No

## Definition of Done
- [x] VFS: Creates issue and provides all required information above
- [ ] VSP: Completes final IA feedback/recommendation
- [ ] VSP: [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886) is updated
- [ ] VFS: Closes issue once all IA feedback/recommendations are implemented

**Comment by mnorthuis** [comment]
@ajakabcin I just realized that I had grouped this with the other profile issue #23286, and was thinking this one was on hold as well.  Can you let me know if I should indeed get this one done, or if you wanted me to wait on this one as well?

**Comment by ajakabcin** [comment]
No worries @mnorthuis - yes this one is good to go too!

**Comment by andaleliz** [comment]
@mnorthuis sorry about that, looks like it got moved. Here's the new link: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/preferred-facility/frontend/design-specs.md

I'll update the description as well.

**Comment by mnorthuis** [comment]
IA for this can be found here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/auth-profile-notifications.md

A few notes: 

- Given this feature of profile is pretty targeted to the VAOS flows, I did not recommend any additional entry points.  An article in Resources and support could be created, but this feature feels pretty specific to the online scheduling user flow, and not necessarily something someone would be seeking outside of that flow.  That said, and R&S article wouldn't hurt anything, it would just create more visibility to it. 
- The VAOS team is most familiar with the various flows within the online scheduling tool. They should propose the best placement of a link to this feature from within their tool.
- For the URL, I kept it simple since 1) this is an authenticated page that does need search optimization 2) I don't know how this feature will grow over time in terms of types of facilities and 3) using "your" in the URL doesn't provide any significant benefit. 

Let me know if you have any questions or concerns. You are welcome to close this issue at your discretion.
